src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -157,6 +180,21 @@ export class KeycloakUserService {
         }
     }
 
+    private async findByPersonId(personId: string): Promise<Result<UserDo<true>, DomainError>> {
+        const person: Result<PersonDo<true>> = await this.personService.findPersonById(personId);
+        if (person.ok) {
+            return this.findById(person.value.keycloakUserId);
+        }
+        return {
+            ok: false,
+            error: new EntityNotFoundError(),
+        };
+    }
+
+    private generatePassword(): string {
+        return faker.string.alphanumeric({ length: { min: 10, max: 10 }, casing: 'mixed' });
```

Comment:
This seems like a misuse of faker. Usually it is used to generate testdata not productive data. I would prefer to use something different to remove the (IMO) testing dependancy from productive code.

Reply:
Do you know something I could use?

Reply:
node has a built in crypto library that can be used for this.

in general it would look like this:
```
import { randomBytes } from 'node:crypto';
.
.
.
randomBytes(10).toString('hex');
```